scripts/galaxykickstart_from_workflow.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+import textwrap
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from shutil import copyfile
+from generate_tool_list_from_ga_workflow_files import *
```

Comment:
`*` imports are strongly discouraged, how about `import generate_tool_list_from_ga_workflow_files as wf_helper` and then referencing the functions as wf_helper.myfunction ?